tests/unit/composables/Project.spec.js

```diff
@@ -57,6 +58,7 @@ jest.mock('isomorphic-git', () => ({
     onAuth();
     return Promise.resolve('pull');
   }),
+  add: jest.fn(() => Promise.resolve('add')),
```

Comment:
this function is suppose to return nothing on success according to the function doc.

Reply:
same for `checkout` function but we also wrote `checkout: jest.fn(() => Promise.resolve('checkout')),` ?

Reply:
done